R/utils-jupyterlab.R

```diff
@@ -15,10 +15,11 @@ py_mimebundle <- function(obj) {
 #'
 #' @return character
 #' @keywords internal
+#' @importFrom repr repr_text
 #' @export
 #'
 repr_text.altair.vegalite.v2.api.Chart <- function(obj) {
-  py_mimebundle(chart)$`text/plain`
+  py_mimebundle(obj)$`text/plain`
```

Comment:
you also forgot to rename chart→obj here

Reply:
Doh... Thanks!